### PR TITLE
feat: Move edit and delete options to toolbar for TracksFragment #963

### LINKED ISSUE #963: Move edit and delete operations to toolbar for TracksFragment
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request
    
- [ ] Chore
        
### Feature Request

**Feature Description**

Edit and delete options from every item in the list should be moved to the toolbar, allowing the user to long press an item to edit or delete it while also supporting multiple deletions.
Edit option should not be present in case of multiple selections.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mishuvs**
Fixes #963 

Changes:

1. moves edit and delete options from every item in the list to the toolbar
2. allows the user to long press an item to edit or delete it
3. supports multiple deletions
4. removes edit option in case of multiple selections
5. re-clicking the last selected item unselects
6. re-clicking the only selected item resets the toolbar

Gif deletion:

videotogif_2018 05 22_22 17 45

Gif Updation:

videotogif_2018 05 20_13 22 24 1

### CODE REVIEW

**Comment by mishuvs** [comment]
@iamareebjamal Please review

**Comment by iamareebjamal** [comment]
Only cardview should be highlighted

Track deleted is shown before the tracks are deleted. No visual progress on deleting

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -27,4 +27,5 @@
     <color name="tint_drawer_item_icon">#8C000000</color>
     <color name="color_drawer_item_text">#D9000000</color>
     <color name="event_state_draft">#FFF9C4</color>
+    <color name="white">#FFFFFF</color>
```
Comment (iamareebjamal) [inline app/src/main/res/values/colors.xml]: Use android white

**Comment by mishuvs** [comment]
@iamareebjamal Please review

**Pull request merged**